### PR TITLE
Fix CLI docker compose

### PR DESCRIPTION
**Pull request by NicolasMahe**
Fix CLI docker compose and switch to image `liteflow/backend:v0.1.0`

### CODE REVIEW

File: packages/cli/src/commands/dev.ts [hunk 1]
```diff
@@ -139,7 +139,7 @@ export default class Dev extends Command {
 
     await stop({
       cwd: join(__dirname, '..', '..', 'assets'),
-      composeOptions: ['-p', 'liteflow'],
+      composeOptions: ['-p', 'liteflow-dev'],
```
Comment (NicolasMahe) [inline packages/cli/src/commands/dev.ts:142]:
```suggestion
      composeOptions: ['-p', 'liteflow_dev'],
```

File: packages/cli/src/commands/dev.ts [hunk 2]
```diff
@@ -95,7 +95,7 @@ export default class Dev extends Command {
     this.engineLogger.info('starting the engine')
     await upAll({
       cwd: join(__dirname, '..', '..', 'assets'),
-      composeOptions: ['-p', 'liteflow'],
+      composeOptions: ['-p', 'liteflow-dev'],
```
Comment (NicolasMahe) [inline packages/cli/src/commands/dev.ts:98]:
```suggestion
      composeOptions: ['-p', 'liteflow_dev'],
```

File: packages/cli/assets/docker-compose.yml [hunk 3]
```diff
@@ -1,33 +1,26 @@
...
       - postgres
     environment:
-      - SCRYPT_PARAMS={}
-      - NODE_ENV=development
-      - DATABASE_HOST=postgres
-      - TYPEORM_HOST=postgres
+      - POSTGRES_HOST=postgres
+      - POSTGRES_USER=dev
+      - POSTGRES_PASSWORD=dev
+      - POSTGRES_DB=dev
```
Comment (antho1404) [inline packages/cli/assets/docker-compose.yml:13]: don't we need the typeorm schema reset env variable as well?

Comment (NicolasMahe) [reply packages/cli/assets/docker-compose.yml:13]: so we reset at each run of dev command? I'm ok with this 👍

Comment (NicolasMahe) [reply packages/cli/assets/docker-compose.yml:13]: done

**Pull request merged**